.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  deploy:
```

Comment:
Regarding my comment about the name, I think if you change this from `deploy` to `test` it will give a recognizable name in the checks.